pkg/utils/utils.go

```diff
@@ -51,6 +55,25 @@ func init() {
 	ClusterType = os.Getenv("CLUSTER_TYPE")
 }
 
+// GetCurrentKernelArgs This retrieves the kernel cmd line arguments
+func GetCurrentKernelArgs(chroot bool) (string, error) {
+	path := procKernelCmdLine
+	if !chroot {
+		path = "/host" + path
+	}
+	cmdLine, err := os.ReadFile(path)
+	if err != nil {
+		return "", fmt.Errorf("GetCurrentKernelArgs(): Error reading %s: %v", procKernelCmdLine, err)
+	}
+	return string(cmdLine), nil
+}
+
+// IsKernelArgsSet This checks if the kernel cmd line is set properly. Please note that the same key could be repeated
+// several times in the kernel cmd line. We can only ensure that the kernel cmd line has the key/val kernel arg that we set.
+func IsKernelArgsSet(cmdLine string, karg string) bool {
+	return strings.Contains(cmdLine, karg)
```

Comment:
I'm wondering, let's say we want to check if "a=b" is set, and we have another already set "ba=a", then is it the case that this will return true? I think we might want to split the string on spaces and than compare in each string.

Reply:
Good catch! I split it in an array + compare.